tools/util.wdl

```diff
@@ -301,12 +279,30 @@ task compression_integrity {
     runtime {
         memory: "~{memory_gb} GB"
         disk: "~{disk_size_gb} GB"
-        docker: 'quay.io/biocontainers/samtools:1.16.1--h6899075_1'
+        docker: 'quay.io/biocontainers/samtools:1.17--h00cdaf9_0'
         maxRetries: max_retries
     }
 }
 
 task add_to_bam_header {
+    # Should this be in samtools.wdl instead of util.wdl?
```

Comment:
I suppose it could go there. However, it isn't a straightforward invocation of a `samtools` command. It serves a more specialized purpose. So to me, that suggests `util` is the place for it.

Reply:
By that argument, we should move `subsample` out of `samtools.wdl` and into `util`. I disagree.

I would include `samtools` in my list of "files that need to be shortened/split up", so between having this task in `util` or `samtools`, I feel bothered either way lol. For simplicity (or you could probably say laziness), let's leave it as is. We can come back to this if my movement to shorten our files gets traction. This is a candidate for moving _somewhere_, I'm just not sure where yet.